nvcodec/README.md

```diff
@@ -0,0 +1,20 @@
+JavaCPP Presets for NVCodec
+============================
+
+[![Gitter](https://badges.gitter.im/bytedeco/javacpp.svg)](https://gitter.im/bytedeco/javacpp) [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.bytedeco/nvcodec/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.bytedeco/nvcodec) [![Sonatype Nexus (Snapshots)](https://img.shields.io/nexus/s/https/oss.sonatype.org/org.bytedeco/nvcodec.svg)](http://bytedeco.org/builds/)
+
+License Agreements
+------------------
+By downloading these archives, you agree to the terms of the license agreements for NVIDIA software included in the archives.
+
+### NVCodec
+To view the license for NVCodec included in these archives, click [here](https://docs.nvidia.com/video-technologies/video-codec-sdk/license/)
+
+
+Introduction
+------------
+This directory contains the JavaCPP Presets module for:
+
+ * NVIDIA Video Codec SDK 11.0.10  https://developer.nvidia.com/nvidia-video-codec-sdk
+
+Please refer to the parent README.md file for more detailed information about the JavaCPP Presets.
```

Comment:
Let's add here "Documentation" and "Sample Usage" sections for consistency.

As sample usage, how about a small "Hello, World" calling NvEncodeAPIGetMaxSupportedVersion() or something?

Reply:
We have to make two part(encoder, decoder) sample

i will make encoder sample use "NvEncodeAPIGetMaxSupportedVersion",

and decoder sample use "cuvidGetDecoderCaps" with MPEG-1 parameters (because, all gpu support mepg-1 codec. so, i will be return always true)

Reply:
Ok, sources good, but include both calls in the same class. It's just for something to show in the README.md file.

Reply:
Yeap, i will write